app/jobs/store_version_contents_job.rb

```diff
@@ -0,0 +1,39 @@
+class StoreVersionContentsJob < ApplicationJob
+  include GoodJob::ActiveJobExtensions::Concurrency
+  good_job_control_concurrency_with total_limit: 1, key: -> { "store-contents-#{version_arg.full_name}" }
+  queue_as :default
```

Comment:
What about scope those 2 new jobs into custom queue like `gem_contents`?

Reply:
Good idea.

Reply:
I used `version_contents` to match the job names.